### ISSUE TITLE
What does the "2'' mean in this line of code? (from model solution Lab 5 Q4)

### ISSUE THREAD

**Issue by dudereal** [chulminy/AE121#39]
word_loc = zeros(numel(word), 2);

**Comment by jpconnel** [comment]
The 2 is the number of columns you want in the matrix of zeros. The dimensions of 'word_loc' are 'numel(word)' rows and 2 columns.

**Comment by dudereal** [comment]
would numel(word) be two numbers  or just a single digit? 
numel(word) is the length of the word so it would just be one number? 

--> what would go in the two columns?

**Comment by jpconnel** [comment]
Exactly. numel(word) returns a scalar number. So when we use it as the first input in the zeros function, we are saying we want 'word_loc' to have 'numel(word)' rows, since the first input of the zeros function is the number of rows of 0's you want.

**Comment by dudereal** [comment]
2 is the second input of the word_loc ?
but what will these two columns contain?

**Comment by jpconnel** [comment]
No. 2 is an input for the zeros function that was used to make 'word_loc'.

Remember how to use zeros: zeros(inp1, inp2) will make a 2D matrix of zeros, with 'inp1' rows and 'inp2' columns.

In the example above, 'inp1' is 'numel(word)' and 'inp2' is 2.

**Comment by dudereal** [comment]
is test_loc a vector ? because ex jj=2 n_word= 6 then it would be 2:7 , therefore test_loc = [ 2 3 4 5 6 7]

also n_word is the length of the word right?

**Comment by chulminy** [comment]
n_word is the size of the array. 

![image](https://user-images.githubusercontent.com/3227103/61591005-a432a900-ab8e-11e9-9e9e-745c3a66fd83.png)

![image](https://user-images.githubusercontent.com/3227103/61591008-af85d480-ab8e-11e9-86ca-2fb8f5f01eb1.png)
